File: zxydialog/src/main/java/com/zxy/zxydialog/TToast.java
package com.zxy.zxydialog;

import android.view.View;

import com.zxy.zxydialog.tools.Applications;
import com.zxy.zxydialog.tools.MyToast;

public class TToast {
    public static void show(String message) {
        MyToast.getINSTANCE().show(message);
    }

    public static void show(int message) {
        MyToast.getINSTANCE().show(Applications.context().getResources().getString(message));
    }

    public static void show(String message, int time) {
        MyToast.getINSTANCE().show(message, time);
    }

    public static void show(int message, int time) {
        MyToast.getINSTANCE().show(Applications.context().getResources().getString(message), time);
    }

    public static void show(View layoutView) {
        MyToast.getINSTANCE().show(layoutView);
    }

    public static void show(View layoutView, int time) {
        MyToast.getINSTANCE().show(layoutView, time);
    }
}

File: zxydialog/src/main/java/com/zxy/zxydialog/TToast.kt
package com.zxy.zxydialog

import android.annotation.SuppressLint
import android.widget.Toast
import com.zxy.zxydialog.tools.Applications
import com.zxy.zxydialog.tools.MyToast

/**
 * Created by zxy on 2020/9/2 11:17
 * ******************************************
 * *
 * ******************************************
 */
class TToast {
    companion object {
        fun show(message: String) {
            MyToast.instance().show(message)
        }

        fun show(message: String, time: Int) {
            MyToast.instance().show(message, time)
        }
    }

}